### PR TITLE
feat: Insight Subscription DB migration

### PR DESCRIPTION
**Pull request by benjackwhite**
Migration only PR from https://github.com/PostHog/posthog/pull/10197

Should be merged before  https://github.com/PostHog/posthog/pull/10197 once that PR is ready.

### CODE REVIEW

File: posthog/models/subscription.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    # Relations - i.e. WHAT are we exporting?
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+    dashboard = models.ForeignKey("posthog.Dashboard", on_delete=models.CASCADE, null=True)
+    insight = models.ForeignKey("posthog.Insight", on_delete=models.CASCADE, null=True)
+
+    # Subscription type (email, slack etc.)
+    title: models.CharField = models.CharField(max_length=100, null=True, blank=True)
+    target_type: models.CharField = models.CharField(max_length=10, choices=SubscriptionTarget.choices)
+    target_value: models.CharField = models.CharField(max_length=65535)
```
Comment (mariusandra) [inline posthog/models/subscription.py:40]: I guess "64k of emails" should be enough for anybody, right? 😅 

Should this be a textfield? Or is this a future problem?

**Review by mariusandra — APPROVED** [review]
One comment inline, but happy to get it in as is.

File: posthog/models/subscription.py (hunk 1 above)
Comment (mariusandra) [reply posthog/models/subscription.py:40]: That makes sense. I'm just thinking if it should be a limitless `TextField`... or a 64k CharField

Comment (benjackwhite) [reply posthog/models/subscription.py:40]: Hmmm thats a good point. I think a TextField is more accurate tbh and less mysteriously specific than the value I gave

**Pull request merged**